bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -249,6 +261,9 @@ public Row readModifyWriteRow(ReadModifyWriteRowRequest request) {
       streamingBufferSize = 10;
       channel = channelPool;
     } else {
+      // This ensures that if there is an ongoing scan, a user can also perform additional RPCs
+      // like delete and get on a second connection.
+      expandPoolIfNecessary(2);
```

Comment:
We only need 2 channels here.  I'm not sure about what we resources we lose with more channels than necessary.  FWIW, expandPoolIfNecessary() is max(currentCount, newCount)